### ISSUE TITLE
instance status_count is returned as string

### ISSUE THREAD

**Issue by technicat** [pixelfed/pixelfed#5178]
I just noticed my code is breaking on reading instance.stats because the status_count is returned as a string instead of an int (per the mastodon spec). I'm guessing it's due to the new call to totalLocalStatuses() in the "update cache" commit from a few days ago (other instances I've checked besides pixelfed.social still return all ints). I added more flexible int-or-string decoding on my end but it may impact other apps.

```json
"stats" : {
    "domain_count" : 25881,
    "user_count" : 115132,
    "status_count" : "26956404"
  },
```

**Comment by JoshHrach** [comment]
Also seeing this with a smaller server I'm using to test an app I'm building with TootSDK. Server is running v0.12.3.

**Comment by JoshHrach** [comment]
I wonder if explicitly casting this to an `Int` will fix the issue.

https://github.com/pixelfed/pixelfed/blob/c33e5b07779a62fc5a3fd7926075c502c1d32d35/app/Services/StatusService.php#L199

**Comment by dansup** [comment]
Fixed in dcd95d68
